### PR TITLE
Overview image gallery

### PR DESCRIPTION
**Pull request by xintongmi**
Bundled several changes at this time (will definetely avoid that in future):
- built first test suite;
- styled overview component;
- refactored style and quantity selector from <select> and <option> to <button> to allow fire event from other components;
- started to build image gallery

### CODE REVIEW

File: client/src/components/overview/Description.jsx
Comment (jtoneill) [inline client/src/components/overview/Description.jsx:1]: I've never seen the ? used in this way. Line 41, 42, 46. Super cool.

File: client/src/components/overview/ImageGallery.jsx [hunk 1]
```diff
@@ -1,21 +1,134 @@
...
+
+const ThumbnailViewContainer = styled.div`
+  position: absolute;
+  top: 45px;
+  left: 10px;
+  width: 90px;
+  height: 600px;
+  display: felx;
+  flex-direction: column;
+  justify-containt: space-between;
```
Comment (jtoneill) [inline client/src/components/overview/ImageGallery.jsx]: I think this is a typo on line 31, justify-content?

File: webpack.config.js
Comment (jtoneill) [inline webpack.config.js:1]: Not sure what this rule is for but I assume you / we need it for tests.

**Review by jtoneill — COMMENTED** [review]
Everything looks good except for that one typo I found in ImageGallery.jsx, line 31

File: client/src/components/overview/ImageGallery.jsx (hunk 1 above)
Comment (xintongmi) [reply client/src/components/overview/ImageGallery.jsx]: Good catch! Thanks! Will update with next commit to this pull request

File: webpack.config.js
Comment (xintongmi) [reply webpack.config.js:1]: With this we can use any png / jpg / svg / gif in our code. I've used a dropdown icon in svg format (tried to just use unicode, but that looks very ugly:(

File: client/src/components/overview/Description.jsx
Comment (xintongmi) [reply client/src/components/overview/Description.jsx:1]: Yup, optional chaining can make the code shorter

**Review by jtoneill — APPROVED** [review]
Looks good!

**Pull request merged**